pkg/providers/vsphere/defaults.go

```diff
@@ -69,7 +73,7 @@ func setDefaultsForEtcdMachineConfig(machineConfig *anywherev1.VSphereMachineCon
 	}
 }
 
-func (d *Defaulter) setDefaultTemplateIfMissing(ctx context.Context, spec *Spec, workerNodeGroup anywherev1.WorkerNodeGroupConfiguration) error {
+func (d *Defaulter) setWorkerDefaultTemplateIfMissing(ctx context.Context, spec *Spec, workerNodeGroup anywherev1.WorkerNodeGroupConfiguration) error {
```

Comment:
can you also update the log on line 83 to say worker node instead of control plane?